ninja_build.sh

```diff
@@ -47,6 +47,7 @@ MUSESCORE_NO_RPATH=${MUSESCORE_NO_RPATH:-"OFF"}
 MUSESCORE_MODULE_UPDATE=${MUSESCORE_MODULE_UPDATEE:-"ON"}
 MUSESCORE_BUILD_VST_MODULE=${MUSESCORE_BUILD_VST_MODULE:-"OFF"}
 MUSESCORE_BUILD_IMPEXP_VIDEOEXPORT_MODULE=${MUSESCORE_BUILD_IMPEXP_VIDEOEXPORT_MODULE:-"OFF"}
+MUSESCORE_BUILD_IMPEXP_LYRICS_MODULE==${MUSESCORE_BUILD_IMPEXP_LYRICS_MODULE:-"OFF"}
```

Comment:
I don't think we need this here; since it's such a small module with no complicated dependencies, it can just be enabled by default.